src/PIL/ImageFile.py

```diff
@@ -63,15 +64,25 @@
 # Helpers
 
 
-def raise_oserror(error):
+def _get_oserror(error, *, encoder=False):
```

Comment:
```suggestion
def _get_oserror(error, *, encoder):
```
Given https://github.com/python-pillow/Pillow/pull/7609/files#r1423830172, maybe requiring the argument would be helpful?

Reply:
Yup, done.